db/dcrsqlite/apisource.go

```diff
@@ -1328,7 +1337,20 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 		KnownTransactions: numberMaxOfTx,
 		KnownFundingTxns:  numReceiving,
 		KnownSpendingTxns: numSpending,
+		IsDummyAddress:    false,
```

Comment:
Please remove.  false is the zero value for booleans used by default during struct init.